sda-commons-shared-wiremock-testing/build.gradle

```
@@ -0,0 +1,10 @@
+dependencies {
+
+  api "org.wiremock:wiremock", {
+    exclude group: 'com.fasterxml.jackson', module: 'jackson-bom'
+  }
+
+  api 'net.minidev:json-smart'
```

I would add an exclude for this dependency in the wiremock dependency to be sure to remove any version conflict.

```suggestion
  api "org.wiremock:wiremock", {
    exclude group: 'com.fasterxml.jackson', module: 'jackson-bom'
    exclude group: 'net.minidev', module: 'json-smart'
  }

  api 'net.minidev:json-smart'

```

---

change added
https://github.com/SDA-SE/sda-dropwizard-commons/pull/2973